webpack.config.js

```diff
@@ -66,46 +75,42 @@ module.exports = function (env, argv) {
       new HtmlWebpackPlugin({
         filename: path.resolve(__dirname, 'out/assets/overview/index.html'),
         template: 'src/overview/assets/index.html',
-        inlineSource: '.(js|css)$',
         chunks: ['overview']
       }),
       new HtmlWebpackPlugin({
         filename: path.resolve(__dirname, 'out/assets/java-runtime/index.html'),
         template: 'src/java-runtime/assets/index.html',
-        inlineSource: '.(js|css)$',
         chunks: ['java-runtime']
       }),
       new HtmlWebpackPlugin({
         filename: path.resolve(__dirname, 'out/assets/getting-started/index.html'),
         template: 'src/getting-started/assets/index.html',
-        inlineSource: '.(js|css)$',
         chunks: ['getting-started']
       }),
       new HtmlWebpackPlugin({
         filename: path.resolve(__dirname, 'out/assets/ext-guide/index.html'),
         template: 'src/ext-guide/assets/index.html',
-        inlineSource: '.(js|css)$',
         chunks: ['ext-guide']
       }),
       new HtmlWebpackPlugin({
         filename: path.resolve(__dirname, 'out/assets/welcome/index.html'),
         template: 'src/welcome/assets/index.html',
-        inlineSource: '.(js|css)$',
         chunks: ['welcome']
       }),
       new HtmlWebpackPlugin({
         filename: path.resolve(__dirname, 'out/assets/classpath/index.html'),
         template: 'src/classpath/assets/index.html',
-        inlineSource: '.(js|css)$',
         chunks: ['classpath']
       }),
       new HtmlWebpackPlugin({
         filename: path.resolve(__dirname, 'out/assets/formatter-settings/index.html'),
         template: 'src/formatter-settings/assets/index.html',
-        inlineSource: '.(js|css)$',
         chunks: ['formatter-settings']
       }),
-      new HtmlWebpackInlineSourcePlugin(),
+      new InlineChunkHtmlPlugin(HtmlWebpackPlugin, [/\.(js)$/]),
+      new webpack.ProvidePlugin({
```

Comment:
In webpack 5 automatic node.js polyfills are removed. Please refer to https://github.com/webpack/changelog-v5#automatic-nodejs-polyfills-removed and https://stackoverflow.com/questions/65018431/webpack-5-uncaught-referenceerror-process-is-not-defined